src/docfx/config/Config.cs

```diff
@@ -107,6 +107,11 @@ internal class Config : PreloadConfig
         /// </summary>
         public int MaxFileInfos { get; private set; } = 20;
 
+        /// <summary>
+        /// Gets the maximum remove deduplication log count
+        /// </summary>
+        public int MaxRemoveDeduplicationLogCount { get; private set; } = 200;
```

Comment:
we don't expect user to change that count, so how about we hard code at where it's used?

Reply:
Got it